app/src/main/java/chat/rocket/android/main/ui/Menu.kt

```diff
@@ -0,0 +1,69 @@
+package chat.rocket.android.main.ui
+
+import android.view.Menu
+import android.view.MenuItem
+import chat.rocket.android.R
+
+internal fun MainActivity.setupMenu(menu: Menu) {
```

Comment:
If this is used only on MainActivity, it could be declared directly there, no gain to be an extension method declared somewhere else

Reply:
Avoiding a long file and taking advantage of decorator pattern by Kotlin native support.